frontend/src/apis/create-api.ts

```diff
@@ -0,0 +1,32 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { UserProfile } from 'models';
+
+enum ApiTag {
+  PROFILE = 'Profile',
+}
+export interface Post {
+  id: string;
+  profile: UserProfile;
+}
+
+// Define a service using a base URL and expected endpoints
+export const api = createApi({
+  baseQuery: fetchBaseQuery({ baseUrl: 'http://127.0.0.1:3002/aurora-instagram-clone/us-central1/api' }),
```

Comment:
You are also using the url string in the `auth.ts` file. In this file, extract this URL into a separate string variable and export it. Then reuse it in the `auth.ts` file.